health/health.c

```diff
@@ -139,6 +139,9 @@ void health_reload_host(RRDHOST *host) {
             t->flags |= HEALTH_ENTRY_FLAG_UPDATED;
     }
 
+    //Reload the section "host labels" from netdata.conf, because it can be new labels that will indicate alarms.
+    config_reload_section(NULL, "host labels");
```

Comment:
Should probably discuss this; to remove spaces from the section

Reply:
It is now the unique section with space, I agree that this is a break of pattern and should be discussed.